pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>org.wikimedia.search.highlighter</groupId>
     <artifactId>experimental</artifactId>
-    <version>7.5.2-SNAPSHOT</version>
+    <version>7.6.1-SNAPSHOT</version>
```

Comment:
this version needs to be updated in all submodule as well

Reply:
@nomoa  thanks for the review comments.